src/lantern/src/JITExecute.cpp

```diff
@@ -0,0 +1,185 @@
+  return make_raw::vector::Argument(arguments);
+  LANTERN_FUNCTION_END_RET(nullptr)
+}
+
+int _lantern_function_schema_num_arguments(void* arglist) {
+  LANTERN_FUNCTION_START
+  const auto& a {from_raw::vector::Argument(arglist)};
+  return a.size();
+  LANTERN_FUNCTION_END_RET(0)
+}
+
+void* _lantern_function_schema_argument_at(void* arglist, int i) {
+  LANTERN_FUNCTION_START
+  const auto& a {from_raw::vector::Argument(arglist)};
+  const auto& at {a.at(i)};
+  return make_raw::Argument(at);
+  LANTERN_FUNCTION_END_RET(nullptr)
+}
+
+void* _lantern_function_schema_argument_name(void* argument) {
+  LANTERN_FUNCTION_START
+  const auto& a {from_raw::Argument(argument)};
+  const auto& argname {a.name()};
+  return make_raw::string(argname);
+  LANTERN_FUNCTION_END_RET(nullptr)
+}
+
+void* _lantern_function_schema_argument_type(void* argument) {
+  LANTERN_FUNCTION_START
+  const auto& a {from_raw::Argument(argument)};
+  const auto& argtype {c10::typeKindToString(a.type()->kind())};
+  return make_raw::string(argtype);
+  LANTERN_FUNCTION_END_RET(nullptr)
+}
+
+void* _lantern_function_schema_returns(void* schema) {
+  LANTERN_FUNCTION_START
+  const auto& s {from_raw::FunctionSchema(schema)};
+  const auto& returns {s.returns()};
+  return make_raw::vector::Argument(returns);
+  LANTERN_FUNCTION_END_RET(nullptr)
+}
+
+void* _lantern_function_schema_return_type(void* return_) {
+  LANTERN_FUNCTION_START
+  const auto& r {from_raw::Argument(return_)};
+  const auto& argtype {c10::typeKindToString(r.type()->kind())};
+  return make_raw::string(argtype);
+  LANTERN_FUNCTION_END_RET(nullptr)
+}
+
+int _lantern_function_schema_list_size(void* schema_list) {
+  LANTERN_FUNCTION_START
+  const auto& s {from_raw::vector::FunctionSchema(schema_list)};
+  return s.size();
+  LANTERN_FUNCTION_END_RET(0)
+}
+
+void* _lantern_function_schema_list_at(void* schema_list, int i) {
+  LANTERN_FUNCTION_START
+  const auto& a {from_raw::vector::FunctionSchema(schema_list)};
+  const auto& at {a.at(i)};
+  return make_raw::FunctionSchema(at);
+  LANTERN_FUNCTION_END_RET(nullptr)
+}
+
+// Return a list of registered operators.
+void* _lantern_jit_all_operators() {
+  LANTERN_FUNCTION_START
+  const auto& ops {torch::jit::getAllOperators()};
+  std::vector<std::string> names;
+  for (const auto& op : ops) {
+    names.push_back(op->schema().name());
+  }
+  return make_raw::vector::string(names);
+  LANTERN_FUNCTION_END_RET(nullptr)
+}
+
+// Obtain the default FunctionSchema for a specific operator.
+// The schema contains information about arguments (name and type) and return types.
+void* _lantern_jit_operator_info(void* name) {
+  LANTERN_FUNCTION_START
+  const auto& op_name {c10::Symbol::fromQualString(from_raw::string(name))};
+  const auto& ops {torch::jit::getAllOperatorsFor(op_name)};
+  const auto length {ops.size()};
+  if (length > 1) {
+    std::cout << "Returning default schema. Call _jit_all_schemas_for() to get all." << std::endl;
```

Comment:
Write to stdout like this is not a good practice as it's very hard for users quiet those messages. I understand it's useful for debugiing, but we can't have it by default when we release it.